test/e2e/case39_diff_generation_test.go

```diff
@@ -210,9 +214,38 @@ var _ = Describe("Diff generation with sensitive input", Ordered, func() {
 
 		diff, _, _ := unstructured.NestedString(relatedObjects[0].(map[string]interface{}), "properties", "diff")
 		Expect(diff).To(Equal(
-			`# This diff may contain sensitive data. The "recordDiff" field must be set to "InStatus" ` +
-				`to record a diff.`,
+			`# The difference is redacted because it contains sensitive data. To override, the ` +
+				`spec["object-templates"][].recordDiff field must be set to "InStatus" for the difference to be ` +
+				`recorded in the policy status. Consider existing access to the ConfigurationPolicy objects and the ` +
+				`etcd encryption configuration before you proceed with an override.`,
 		))
+
+		By("Enforcing the policy removes the diff message")
+		utils.Kubectl(
+			"patch", "configurationpolicy", noDiffOnSecret, `--type=json`,
+			`-p=[{"op":"replace","path":"/spec/remediationAction","value":"enforce"}]`, "-n", testNamespace,
+		)
+
+		By("Verifying the diff in the status no longer contains instructions to set recordDiff")
```

Comment:
```suggestion
		By("Verifying the difference in the status no longer contains instructions to set recordDiff")
```

Reply:
This is a log message in the tests, so this doesn't need changes.